### PR TITLE
chore(docs): add missing dnf command and additional notes on docker

### LINKED ISSUE #43: Fix typo in troubleshooting section of README
`README.md` mentions:

https://github.com/cryostatio/cryostat-reports/blob/09305e3a11b7aaf8591d8b9ded629347c4268af2/README.md?plain=1#L41

This should be `sudo dnf install podman-docker` (maybe other package manager?). Also, I think installing and starting `docker` would also solve this issue.

### PR DESCRIPTION
**Pull request by tthvo**
Fixes #43 

Add missing `dnf` command and additional notes for cases with installed docker.

### CODE REVIEW

**Comment by tthvo** [comment]
With `docker` installed via moby-engine, `sudo dnf install podman-docker` gives conflict errors:

```bash
Error: 
 Problem: problem with installed package moby-engine-20.10.17-4.fc35.x86_64
  - package moby-engine-20.10.17-4.fc35.x86_64 conflicts with podman-docker provided by podman-docker-3:3.4.0-1.fc35.noarch
  - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with docker provided by moby-engine-20.10.17-4.fc35.x86_64
  - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with moby-engine provided by moby-engine-20.10.17-4.fc35.x86_64
  - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with docker-latest provided by moby-engine-20.10.17-4.fc35.x86_64
  - package moby-engine-20.10.8-1.fc35.x86_64 conflicts with podman-docker provided by podman-docker-3:3.4.0-1.fc35.noarch
  - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with docker provided by moby-engine-20.10.8-1.fc35.x86_64
  - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with moby-engine provided by moby-engine-20.10.8-1.fc35.x86_64
  - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with docker-latest provided by moby-engine-20.10.8-1.fc35.x86_64
  - conflicting requests
  - package moby-engine-20.10.17-4.fc35.x86_64 conflicts with podman-docker provided by podman-docker-3:3.4.7-2.fc35.noarch
  - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with docker provided by moby-engine-20.10.17-4.fc35.x86_64
  - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with moby-engine provided by moby-engine-20.10.17-4.fc35.x86_64
  - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with docker-latest provided by moby-engine-20.10.17-4.fc35.x86_64
  - package moby-engine-20.10.8-1.fc35.x86_64 conflicts with podman-docker provided by podman-docker-3:3.4.7-2.fc35.noarch
  - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with docker provided by moby-engine-20.10.8-1.fc35.x86_64
  - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with moby-engine provided by moby-engine-20.10.8-1.fc35.x86_64
  - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with docker-latest provided by moby-engine-20.10.8-1.fc35.x86_64
(try to add '--allowerasing' to command line to replace conflicting packages or '--skip-broken' to skip uninstallable packages)
```

**Comment by andrewazores** [comment]
> With `docker` installed via moby-engine, `sudo dnf install podman-docker` gives conflict errors:
> 
> ```shell
> Error: 
>  Problem: problem with installed package moby-engine-20.10.17-4.fc35.x86_64
>   - package moby-engine-20.10.17-4.fc35.x86_64 conflicts with podman-docker provided by podman-docker-3:3.4.0-1.fc35.noarch
>   - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with docker provided by moby-engine-20.10.17-4.fc35.x86_64
>   - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with moby-engine provided by moby-engine-20.10.17-4.fc35.x86_64
>   - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with docker-latest provided by moby-engine-20.10.17-4.fc35.x86_64
>   - package moby-engine-20.10.8-1.fc35.x86_64 conflicts with podman-docker provided by podman-docker-3:3.4.0-1.fc35.noarch
>   - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with docker provided by moby-engine-20.10.8-1.fc35.x86_64
>   - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with moby-engine provided by moby-engine-20.10.8-1.fc35.x86_64
>   - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with docker-latest provided by moby-engine-20.10.8-1.fc35.x86_64
>   - conflicting requests
>   - package moby-engine-20.10.17-4.fc35.x86_64 conflicts with podman-docker provided by podman-docker-3:3.4.7-2.fc35.noarch
>   - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with docker provided by moby-engine-20.10.17-4.fc35.x86_64
>   - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with moby-engine provided by moby-engine-20.10.17-4.fc35.x86_64
>   - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with docker-latest provided by moby-engine-20.10.17-4.fc35.x86_64
>   - package moby-engine-20.10.8-1.fc35.x86_64 conflicts with podman-docker provided by podman-docker-3:3.4.7-2.fc35.noarch
>   - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with docker provided by moby-engine-20.10.8-1.fc35.x86_64
>   - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with moby-engine provided by moby-engine-20.10.8-1.fc35.x86_64
>   - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with docker-latest provided by moby-engine-20.10.8-1.fc35.x86_64
> (try to add '--allowerasing' to command line to replace conflicting packages or '--skip-broken' to skip uninstallable packages)
> ```

That's normal/expected. `podman-docker` is a package that just installs some wrapper scripts (maybe even just symlinks) that let you invoke `docker` as a command but actually run `podman`, so you can only have one package or the other.

File: README.md [hunk 1]
```diff
@@ -38,9 +38,11 @@ and loaded into your local `podman` image registry.
 
 If a Quarkus build failure is encountered due to being unable to build a Docker image, then:
 ```shell script
-sudo install podman-docker
+sudo dnf install podman-docker
 ```
 
+**Note**: If `docker` is already installed, then starting `docker` will solve the issue.
```
Comment (andrewazores) [inline README.md:44]: "starting the `docker` service` perhaps?

**Comment by tthvo** [comment]
> > With `docker` installed via moby-engine, `sudo dnf install podman-docker` gives conflict errors:
> > ```shell
> > Error: 
> >  Problem: problem with installed package moby-engine-20.10.17-4.fc35.x86_64
> >   - package moby-engine-20.10.17-4.fc35.x86_64 conflicts with podman-docker provided by podman-docker-3:3.4.0-1.fc35.noarch
> >   - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with docker provided by moby-engine-20.10.17-4.fc35.x86_64
> >   - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with moby-engine provided by moby-engine-20.10.17-4.fc35.x86_64
> >   - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with docker-latest provided by moby-engine-20.10.17-4.fc35.x86_64
> >   - package moby-engine-20.10.8-1.fc35.x86_64 conflicts with podman-docker provided by podman-docker-3:3.4.0-1.fc35.noarch
> >   - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with docker provided by moby-engine-20.10.8-1.fc35.x86_64
> >   - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with moby-engine provided by moby-engine-20.10.8-1.fc35.x86_64
> >   - package podman-docker-3:3.4.0-1.fc35.noarch conflicts with docker-latest provided by moby-engine-20.10.8-1.fc35.x86_64
> >   - conflicting requests
> >   - package moby-engine-20.10.17-4.fc35.x86_64 conflicts with podman-docker provided by podman-docker-3:3.4.7-2.fc35.noarch
> >   - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with docker provided by moby-engine-20.10.17-4.fc35.x86_64
> >   - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with moby-engine provided by moby-engine-20.10.17-4.fc35.x86_64
> >   - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with docker-latest provided by moby-engine-20.10.17-4.fc35.x86_64
> >   - package moby-engine-20.10.8-1.fc35.x86_64 conflicts with podman-docker provided by podman-docker-3:3.4.7-2.fc35.noarch
> >   - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with docker provided by moby-engine-20.10.8-1.fc35.x86_64
> >   - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with moby-engine provided by moby-engine-20.10.8-1.fc35.x86_64
> >   - package podman-docker-3:3.4.7-2.fc35.noarch conflicts with docker-latest provided by moby-engine-20.10.8-1.fc35.x86_64
> > (try to add '--allowerasing' to command line to replace conflicting packages or '--skip-broken' to skip uninstallable packages)
> > ```
> 
> That's normal/expected. `podman-docker` is a package that just installs some wrapper scripts (maybe even just symlinks) that let you invoke `docker` as a command but actually run `podman`, so you can only have one package or the other.

Ahh make sense!

**Review by andrewazores — APPROVED** [review]

**Pull request merged**